CHANGELOG.md

```diff
@@ -5,12 +5,21 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Updated dependencies for PureScript 0.14 (#17)
+- Fixed the `Semigroup` instance for `Foreign.Object.Object` (#19)
 
 New features:
+- Added roles declarations to allow safe coercions (#16)
```

Comment:
This is a breaking change, because the syntax didn't exist pre-0.14

Reply:
Actually -- @kl0tl, @hdgarrood, should this be considered a breaking change? I guess it _wouldn't_ be breaking if this were already on 0.14, but it's breaking from the perspective of 0.13.8. Maybe this is the correct location after all.

Reply:
This will be relevant for many of the `contrib` libraries, too.

Reply:
This is only breaking in the sense that it requires us to drop support for older compiler versions, so if it’s already noted that we have dropped support for older versions, then I think it’s ok to regard this as a new feature.

Reply:
Incidentally if we were already on 0.14 it would be a breaking change, because it would mean that some (invalid) coercions which previously compiled would start being rejected by the compiler, as Map’s inferred roles would be `representational representational`, so you could eg coerce a map’s key type from `Int` to `Down Int`.